src/ec/packages/ec-component-accordion/accordion.html.twig

```diff
@@ -44,38 +44,37 @@
 
 {# Print the result #}
 
-<dl
+<div
   class="{{ _css_class }}"{{ _extra_attributes|raw }}
   data-ecl-accordion="true"
-  role="presentation"
 >
   {% if _items is not empty %}
-    {% for item in _items %}
-      <dt aria-level="{{ item.level }}" role="heading">
+    {% for _item in _items %}
+      <h{{ _item.level }} class="ecl-accordion__title">
```

Comment:
First of all, thank you for the good refactoring here!
I'm wondering if it would make sense to transfer also the default level of 3 if none is provided? (reference of thought https://github.com/ec-europa/europa-component-library/pull/1039/files#diff-e9f12bf7ac8b4cb687c60a0b2f65ec7aR8)